tests/test_bot.py

```diff
@@ -2574,6 +2575,17 @@ async def make_assertion(url, request_data: RequestData, *args, **kwargs):
 
         await offline_bot.remove_chat_verification(1234)
 
+    async def test_get_my_star_balance(self, offline_bot, monkeypatch):
```

Comment:
did you have a specific reason to add this test in addition to the one below? At least at first glance I don't see anything that the below test doesn't already cover 🤔

Reply:
I just felt that we should have an offline test too and not only depend on the online tests. For e.g. the online test can flaky fail and xpass.